src/test/java/org/jabref/preferences/VersionPreferencesTest.java

```diff
@@ -0,0 +1,21 @@
+package org.jabref.preferences;
+
+import org.jabref.logic.util.Version;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class VersionPreferencesTest {
+
+    @Test
+    public void givenNothingWhenCreatingThenNothingThrown() {
+        new VersionPreferences(Version.parse(""));
```

Comment:
Here the same again. Nice that the test succeeds, but what do we learn from that?

Reply:
I understand that you are missing learnings from this simple test. At least for regression testing or in case of refactoring, this test may already be helpful. In future, more complex test methods may be added to this class.

Reply:
What @lynyus (probably) meant is the following: you usually write a test to verify a certain desired behavior and not to proof the absence of an undesired one.

To make this more concrete, consider the difference between the test method you added and
````java
public void isEmptyForEmptyVersion() {
     versionPreferences = new VersionPreferences(new Version());
     assertEqual(new Version(), versionPreferences.getIgnoredVersion());
}
````
Both tests would fail if the constructor of `VersionPreferences` throws an exception for some reason. However, this new version of the test also fails in case the constructor does not work as expected, namely if the version is not stored in the `ignoredVersion` field.

That being said, normally you don't need to write tests for simple constructors and properties. If you see this as a learning opportunity, you are of course welcome to write such tests.